components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -634,4 +634,4 @@ export function createGitpodService<C extends GitpodClient, S extends GitpodServ
         }
     });
     return new GitpodServiceImpl<C, S>(gitpodServer, { onReconnect });
-}
\ No newline at end of file
+}
```

Comment:
Unless you explicitly want a review from team meta on this, please revert this change.